### PR TITLE
chore: Silence Rollup's `this is undefined` warnings

### LINKED ISSUE #682: The 'this' keyword is equivalent to 'undefined'
There is a noisy warning that occurs if the bundle imports a module which is compiled using TypeScript's `tsc`:

```
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
```

[The warning is mostly harmless](https://github.com/rollup/rollup/issues/794), so it would be nice to have the [option of suppressing it.](https://github.com/rollup/rollup/issues/794#issuecomment-270803587)

### PR DESCRIPTION
**Pull request by rschristian**
Closes #682

Re:https://github.com/developit/microbundle/issues/682#issuecomment-1221769448, the warning is silenced globally.

I'm a little apprehensive about not limiting this to `node_modules/` though, as it could warn of a legitimate issue.

### CODE REVIEW

**Comment by NikolayRn** [comment]
Can you merge it please? @rschristian

**Comment by rschristian** [comment]
Sorry, that's more up to @developit, would need an approval first.

**Review by developit — APPROVED** [review]

**Comment by developit** [comment]
Sorry for the delay, this seems fine to me.

**Pull request merged**
